### PR TITLE
Update description of release notes contribution guide

### PR DESCRIPTION
**Pull request by mathias-baumann-sociomantic**

### CODE REVIEW

File: doc/library-contributor.rst [hunk 1]
```diff
@@ -30,14 +30,19 @@ The ``neptune-release`` tool automatically collates release notes when making a
 release. To make this possible, contributors must follow the following
 guidelines.
 
-Major and Minor Releases
-........................
-
 When making a commit to a Neptune-versioned library, API-affecting changes
```
Comment (gavin-norman-sociomantic) [inline doc/library-contributor.rst]: Suggest:
```
The ``neptune-release`` tool automatically collates release notes when making a release. To make this possible, contributors must write release notes in a specific format (described below), and place them in the ``relnotes`` folder. When the corresponding branch is released, the files in that folder will be collated to form the notes for the release.

The following procedure should be followed:
... (as before)

Patch Releases
..............
For patch releases, a list of all fixed issues will additionally be appended to the release notes. The title of the bug fix pull request is used for this, so it is important to choose a clear and descriptive name for PRs containing bug fixes.
```

Comment (mathias-baumann-sociomantic) [reply doc/library-contributor.rst]: When you write "as before" do you mean as before my PR or as IN my PR?

Comment (gavin-norman-sociomantic) [reply doc/library-contributor.rst]: In you PR. (Sorry, I was forgetting that you'd changed one line in that section.)

**Comment by mathias-baumann-sociomantic** [comment]
Updated

**Comment by mathias-baumann-sociomantic** [comment]
Superseeded by https://github.com/sociomantic-tsunami/neptune/pull/221

**Pull request closed without merging**